web/satellite/tests/unit/project/__snapshots__/ProjectDashboard.spec.ts.snap

```diff
@@ -2,7 +2,12 @@
 
 exports[`ProjectDashboard.vue renders correctly 1`] = `
 <div class="dashboard-area">
-  <h1 class="dashboard-area__title">test Dashboard</h1>
+  <div class="dashboard-area__header-wrapper">
+    <h1 class="dashboard-area__title">test Dashboard</h1>
+    <vinfo-stub text="" boldtext="Expect an hour delay between network activity and the latest dashboard stats." class="dashboard-area__tooltip__wrapper">
```

Comment:
Same as above, lets be more general here.

Reply:
same comment as above